src/teamwork.sh

```diff
@@ -32,21 +32,57 @@ teamwork::add_comment() {
   log::message "$response"
 }
 
+teamwork::add_tag() {
+  local -r tag_name=$1
+
+  if [ "$ENV" == "test" ]; then
+    log::message "Test - Simulate request. Task ID: $TEAMWORK_TASK_ID - Tag: ${tag_name//\"/}"
+    return
+  fi
+
+  response=$(curl -X "PUT" "$TEAMWORK_URI/projects/api/v1/tasks/$TEAMWORK_TASK_ID/tags.json" \
+       -u "$TEAMWORK_API_TOKEN"':' \
+       -H 'Content-Type: application/json; charset=utf-8' \
+       -d "{ \"tags\": { \"content\": \"${tag_name//\"/}\" } }" )
+
+  log::message "$response"
+}
+
+teamwork::remove_tag() {
+  local -r tag_name=$1
+
+  if [ "$ENV" == "test" ]; then
+    log::message "Test - Simulate request. Task ID: $TEAMWORK_TASK_ID - Tag: ${tag_name//\"/}"
```

Comment:
This line is for testing, I would add to the message something like "Removed tag".